features/speech-synthesis.yml

```
@@ -1,5 +1,50 @@
 name: Speech synthesis
-description: The `SpeechSynthesis` API converts text into audio using a synthetic voice.
+description: The `SpeechSynthesis` API converts text into audio using a synthetic voice, with the ability to control various aspects of the interaction and output, including specifcying custom voices.
```

Suggestions for brevity and to avoid a circular description

```suggestion
description: The `SpeechSynthesis` API converts text to speech from artificial voices.
```

---

I find "to speech from artificial voices" confusing, as the "from" is implicitly "text" already- would "to speech with artificial voices" work?